lighthouse-cli/test/smokehouse/test-definitions/oopif/oopif-expectations.js

```diff
@@ -33,7 +33,7 @@ module.exports = [
         {
           id: 'oopif',
           src: 'https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/',
-          clientRect: {
```

Comment:
I think we wanted to preserve this assertion as it was for backcompat?

fine to also add your new one if you'd like :)

Reply:
good catch, you're right!